extensions/amp-analytics/0.1/test/test-transport.js

```diff
@@ -24,22 +24,31 @@ adopt(window);
 describe('amp-analytics.transport', () => {
 
   let sandbox;
+  const transport = new Transport();
   beforeEach(() => {
     sandbox = sinon.sandbox.create();
   });
 
   afterEach(() => {
+    Transport.crossDomainIframes_ = {};
```

Comment:
Rather than accessing private field, create a reset function in Transport with @visibleForTesting